prime-router/src/main/kotlin/azure/ReportFunction.kt

```diff
@@ -140,6 +138,11 @@ class ReportFunction(
         val httpStatus: HttpStatus =
             try {
                 val options = Options.valueOfOrNone(optionsText)
+                if (Options.isDeprecated(options)) {
```

Comment:
Not related to this PR, but can there be multiple options specified???

Reply:
I don't think so which I find very confusing. It seemed out of scope to rename Options to Option for this commit but was a thought I had